src/main/java/org/webonise/producer_consumer/Application.java

```diff
@@ -0,0 +1,30 @@
+package org.webonise.producer_consumer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//here we are having one producer and two consumers.
+public class Application {
+   public static void main(String args[]) {
+      //this is queue which is shared among the threads.
+      List<Integer> sharedQueue = new ArrayList<>();
+
+      //this is the size of queue
+      final int size = 4;
```

Comment:
This can be declared as  public static final in some file, as same value is being used in multiple places